### ISSUE TITLE
Resetvars does not reset variables.

### ISSUE THREAD

**Issue by shacknetisp** [red-eclipse/base#278]
/resetvars does nothing but display the message, in stable client and server.
I tried varslock 0 and 4 with admin auth and someone else tried on varslock 0 with player auth.

**Comment by freezurbern** [comment]
```
[21:55:31] <frzrRE-Gea076bb_> [Ghosts]::Beha (76.179.122.100) has joined the game (global administrator: snisp) [1.5.2-nix64] (1 player)
[21:55:34] <frzrRE-Gea076bb_> [Ghosts]::Beha has map crc: 0x3178ff2d (server: 0x3178ff2d)
[21:55:39] <frzrRE-Gea076bb_> [Ghosts]::Beha set botlimit to 64 (was: 32)
[21:55:41] <frzrRE-Gea076bb_> [Ghosts]::Beha executed resetvars
[21:55:44] <frzrRE-Gea076bb_> [Ghosts]::Beha set botlimit to 32 (was: 64)
[21:55:44] <frzrRE-Gea076bb_> [Ghosts]::Beha (76.179.122.100) has left the game (normal, 0 players)
```

**Comment by qreeves** [comment]
Fixed in 557ec08
